terracumber_config/tf_files/SUSEManager-4.0-qam.tf

```diff
@@ -167,7 +167,7 @@ module "base3" {
 module "server" {
   source             = "./modules/server"
   base_configuration = module.base.configuration
-  product_version    = "4.0-nightly"
+  product_version    = "4.0-released"
```

Comment:
Good catch! Yes, for QAM we should use the released version and inject the latest MI to be validated.